homeassistant/components/plex/media_browser.py

```diff
@@ -249,17 +278,61 @@ def build_item_response(payload):
     return response
 
 
+def generate_plex_uri(server_id, media_id):
+    """Create a media_content_id URL for playable Plex media."""
+    if isinstance(media_id, int):
+        media_id = str(media_id)
+    if isinstance(media_id, str) and not media_id.startswith("/"):
+        media_id = f"/{media_id}"
+    return str(
+        URL.build(
+            scheme=DOMAIN,
+            host=server_id,
+            path=media_id,
+        )
+    )
+
+
+def root_payload(hass, is_internal, platform=None):
+    """Return root payload for Plex."""
+    children = []
+
+    for server_id in hass.data[DOMAIN][SERVERS]:
+        children.append(
+            browse_media(
+                hass,
+                is_internal,
+                "server",
+                generate_plex_uri(server_id, ""),
+                platform=platform,
+            )
+        )
+
+    if len(children) == 1:
+        return children[0]
```

Comment:
This is not correct, as `children[0]` won't have children loaded? This needs to be resolved.

Reply:
Not sure I follow. This list will contain a single `BrowseMedia` instance in this case which represents the only available Plex server.

Reply:
When you browse media, you always return a parent with children. All BrowseMedia instances. If you decide to only return a child, it will not contain it's own children.

Reply:
This just skips the Plex server/integration selection level if only one server is available. For example, if there are two Plex servers configured, you'll be able to choose which one to browse into:
![image](https://user-images.githubusercontent.com/1203111/159178979-1ee6caa1-b26f-46b2-96f8-f20f333c0a75.png)

If only one is configured (or you browse into one of the instances above) you'll skip the above step and go straight into the Plex server contents:
![image](https://user-images.githubusercontent.com/1203111/159179010-d06cb554-535c-436d-9289-732480b832fb.png)